packages/braid-design-system/scripts/generateSnippets.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
```

Comment:
This file should be `.cts` but I discovered our ESLint config needs to be updated to lint files with `.cts`, `.cjs`, `.mjs` and `.mts` extensions.